README.md

```diff
@@ -15,20 +15,27 @@ See [Spring Boot Database Initialization Documentation](https://docs.spring.io/s
 ## Environment Variables
 
 ### .env File Creation
-1. Create an .env file in your backend root directory.
-2. Assign your Database Name, Database Username, and Database Password values to the
-respective variables. Mine looks like this:
+1. Create an .env file in your backend root directory with this information:
 
 ```
+# Assign values to these three variables with information from your local Postgres DB installation:
 POSTGRES_DB=p2team1
 POSTGRES_USER=p2team1_user
 POSTGRES_PASSWORD=password
 
 SPRING_DATASOURCE_USERNAME=${POSTGRES_USER}
 SPRING_DATASOURCE_PASSWORD=${POSTGRES_PASSWORD}
 SPRING_DATASOURCE_URL=jdbc:postgresql://localhost:5432/${POSTGRES_DB}
+# Not entirely sure what this does, yet.  Could change.
```

Comment:
its literally just shorthand for the API URL in the app, you could hardcode this if you want. Its only used in the DataInitializer rn